tests/unit/test_issues_milestone.py

```diff
@@ -50,14 +52,14 @@ def test_none_creator(self):
         """Show that creator is None when json attribute is set to None."""
         json = self.instance.as_dict().copy()
         json['creator'] = None
-        milestone = github3.issues.milestone.Milestone(json)
+        milestone = github3.issues.milestone.Milestone(json, session)
```

Comment:
There's a `create_session` method on our UnitHelper classes. Please use that. It ensures we never accidentally talk to the internet in our unit tests.

Reply:
Cool, I was very unsure about these bits in tests. Glad to have your input 👍